io.edgehog.devicemanager.CellularConnectionStatus.json

```diff
@@ -0,0 +1,58 @@
+{
+  "interface_name": "io.edgehog.devicemanager.CellularConnectionStatus",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "ownership": "device",
+  "aggregation": "object",
+  "mappings": [
+    {
+      "endpoint": "/%{id}/carrier",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Connectivity carrier operator name."
+    },
+    {
+      "endpoint": "/%{id}/cellId",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "The Cell ID in hexadecimal format, either 16 bit for 2G or 28 bit for 3G or 4G."
+    },
+    {
+      "endpoint": "/%{id}/localAreaCode",
+      "type": "string",
+      "database_retention_policy": "use_ttl",
+      "database_retention_ttl": 5184000,
+      "explicit_timestamp": true,
+      "description": "Two byte location area code in hexadecimal format."
+    },
```

Comment:
it looks like we are missing MMC and MNC. See also: https://developers.google.com/maps/documentation/geolocation/overview?hl=tr-tr